components/formats-gpl/src/loci/formats/in/ScreenReader.java

```diff
@@ -78,6 +78,7 @@
   // -- Fields --
 
   private String[] plateMetadataFiles;
+  private String[] excludeReaders;
```

Comment:
The solution in general makes sense, but the introduction of a new field can often cause problems. (Worth testing)